src/main/java/com/codingnomads/betty/logic/services/TwitterQueryConstructor.java

```diff
@@ -0,0 +1,44 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.interfaces.TwitterKeywordRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TwitterQueryConstructor {
```

Comment:
Annotation  maybe ? :)

Reply:
Oh wait it's not a service actually. This was a constructor class that would only be used by the twittersourcerepository class, so i think it should be fine the way it is now. :)